modin/test/test_partition_api.py

```diff
@@ -19,14 +19,34 @@
 from modin.distributed.dataframe.pandas import unwrap_partitions, from_partitions
 from modin.config import Engine, NPartitions
 from modin.pandas.test.utils import df_equals
+from modin.pandas.indexing import compute_sliced_len
+from modin.data_management.factories.dispatcher import EngineDispatcher
 
+PartitionClass = (
+    EngineDispatcher.get_engine().io_cls.frame_cls._partition_mgr_cls._partition_class
+)
 
 if Engine.get() == "Ray":
     import ray
-if Engine.get() == "Dask":
+
+    put_func = ray.put
+    get_func = ray.get
+    FutureType = ray.ObjectRef
+elif Engine.get() == "Dask":
     from distributed.client import get_client
+    from distributed import Future
+
+    put_func = lambda x: get_client().scatter(x)  # noqa: E731
+    get_func = lambda x: x.result()  # noqa: E731
+    FutureType = Future
+else:
```

Comment:
I would rather we explicitly check for supported engine here (this leaves us with `Python` only for now) and raise an exception in `else` branch so newly added engines do not slip tested in a strange way.

Reply:
makes sense. applied.